server/src/server.ts

```diff
@@ -23,9 +24,11 @@ import { WorkspaceEdit } from "vscode-languageserver";
 
 interface extensionConfiguration {
   askToStartBuild: boolean;
+  binaryPath: string;
 }
 let extensionConfiguration: extensionConfiguration = {
   askToStartBuild: true,
+  binaryPath: "",
```

Comment:
This is OK but perhaps a little fragile. How about using null here instead of "".

Reply:
done